editor/src/components/editor/remix-navigation-bar.tsx

```diff
@@ -0,0 +1,110 @@
+import { useAtom } from 'jotai'
+import React from 'react'
+import {
+  ActiveRemixSceneAtom,
+  RemixNavigationAtom,
+} from '../canvas/remix/utopia-remix-root-component'
+import { Substores, useEditorState } from './store/store-hook'
+import { FlexRow, Tooltip, UtopiaTheme, useColorTheme } from '../../uuiui'
+import { stopPropagation } from '../inspector/common/inspector-utils'
+import * as EP from '../../core/shared/element-path'
+
+export const RemixNavigationBarPathTestId = 'remix-navigation-bar-path'
+
+export const RemixNavigationBarHomeLabel = '(home)'
+
+export type RemixSceneLabelButtonType = 'back' | 'forward' | 'home'
+
+export const RemixNavigationBarButtonTestId = (button: RemixSceneLabelButtonType): string =>
+  `remix-navigation-bar-button-${button}`
+
+export const RemixNavigationBar = React.memo(() => {
+  const [navigationControls] = useAtom(RemixNavigationAtom)
+  const [activeRemixScene] = useAtom(ActiveRemixSceneAtom)
+
+  const isLiveMode = useEditorState(
+    Substores.restOfEditor,
+    (_) => _.editor.mode.type === 'live',
```

Comment:
Please don't use the underscore here - we typically use an underscore prefix to indicate a param that is deliberately unused